src/embed_tests/Codecs.cs

```diff
@@ -6,28 +6,34 @@ namespace Python.EmbeddingTest {
     using Python.Runtime;
     using Python.Runtime.Codecs;
 
-    public class Codecs {
+    public class Codecs
```

Comment:
my IDE keeps adding this automatically.  Is there something we can do in visual studio to prevent that?

Reply:
Keep the new formatting. This code was written before I made .editorconfig with the style of braces the rest of the project uses.